open_academy/wizards/attendee_wizards.py

```diff
@@ -0,0 +1,16 @@
+from odoo import fields, models
+
+
+class Attendee(models.TransientModel):
+    _name = "attendee_wizards"
```

Comment:
```suggestion
    _name = "attendee.wizard"
```
change all it's references